examples/flows/standard/web_classification/README.md

```diff
@@ -0,0 +1,128 @@
+# Web Classification
```

Comment:
why do you need to add a new flow example folder?

Reply:
flow in pipeline will use the name of flow directory as component name, but `web-classification` is not a valid component name. Not sure if we should do a automatic transformation in `azure-ai-ml`

Reply:
or let me always set name in current example